### PR TITLE
PBM-1226 Documented custom log path configuration

### PR DESCRIPTION
**Pull request by nastena1606**
new file:   docs/manage/logpath.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/manage/logpath.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Output pbm-agent logs to a file
+
+!!! admonition "Version added: [2.0.0](../release-notes/2.0.0.md)"
+
+
+By default, each `pbm-agent` writes its logs to the following paths:
+
+* the `admin.pbmLogs` PBM Control collection which serves as the centralized logging storage
+* the `stderr` file on the host where the agent is running.
```
Comment (sandraromanchenko) [inline docs/manage/logpath.md:9]:
```suggestion
* `STDERR` stream on the each host where the agent is running
```

File: docs/manage/logpath.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+	```{.bash data-prompt="$"}
+	$ pbm-agent --log-path=/var/log/pbm-agent.log --log-level=W --log-output=json
+	```
+
+!!! note
+
+    Command line flags take precedence over environment variables.
+
+If PBM cannot write to the specified file due to some error, it falls back to the `stderr` file. And you can always retrieve the logging information using the `pbm logs` command.
```
Comment (sandraromanchenko) [inline docs/manage/logpath.md:49]:
```suggestion
If PBM cannot write to the specified file due to some error, it falls back to `STDERR` stream. And you can always retrieve the logging information using the `pbm logs` command.
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: docs/manage/logpath.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Output pbm-agent logs to a file
+
+!!! admonition "Version added: [2.0.0](../release-notes/2.8.0.md)"
```
Comment (nastena1606) [inline docs/manage/logpath.md:3]:
```suggestion
!!! admonition "Version added: [2.8.0](../release-notes/2.8.0.md)"
```

**Review by sandraromanchenko — APPROVED** [review]

File: docs/manage/logpath.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+* simplify the log collection process for further analysis
+* comply with the logging and auditing requirements by storing logs in a secure location for the required period
+
+You can define a logging configuration either via the command line flags or via environment variables when you start a `pbm-agent`. The options are:
+
+| Environment variable | Command line flag | Description | 
+|----------------------|-------------------|-------------|
+| `LOG_PATH` | `--log-path` | The path to the log file. The file is created if it doesn't exist. The default value is `/dev/stderr` which means that the logs are written to the standard error output. |
+| `LOG_LEVEL` | `--log-level` | The log severity level. Supported levels are (from low to high): D - Debug, I - Info (default), W - Warning, E - Error, F - Fatal.<br><br> The output includes both the specified severity level and all higher ones |
+| `LOG_OUTPUT`| `--log-output` | The output format of the log messages. Supported values are `text` (default) and `json`. |
```
Comment (boris-ilijic) [inline docs/manage/logpath.md:24]:
```suggestion
| `LOG_JSON`| `--log-json` | The output format of the log messages. If specified `json` format is use and default is `text` format. |
```

File: docs/manage/logpath.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+	$ export LOG_PATH=/var/log/pbm-agent.log
+	$ export LOG_LEVEL=W
+	$ export LOG_OUTPUT=json
+	$ pbm-agent
+	```
+
+=== ":material-console: Command line"
+
+	```{.bash data-prompt="$"}
+	$ pbm-agent --log-path=/var/log/pbm-agent.log --log-level=W --log-output=json
```
Comment (boris-ilijic) [inline docs/manage/logpath.md:42]:
```suggestion
	$ pbm-agent --log-path=/var/log/pbm-agent.log --log-level=W --log-json
```

**Review by boris-ilijic — COMMENTED** [review]
I am fine with description, but please see comments.

File: docs/manage/logpath.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+## Example
+
+Start the `pbm-agent` on every node with the following command:
+
+=== ":material-variable: Environment variables"
+
+	```{.bash data-prompt="$"}
+	$ export LOG_PATH=/var/log/pbm-agent.log
+	$ export LOG_LEVEL=W
+	$ export LOG_json
```
Comment (boris-ilijic) [inline docs/manage/logpath.md:35]:
```suggestion
	$ export LOG_JSON=1
```

**Review by boris-ilijic — APPROVED** [review]
Please see additonal one

**Pull request merged**